Resources/Private/Partials/Media/VideoTag.html

```diff
@@ -41,5 +41,11 @@
             </p>
         </f:if>
     </div>
+
+    <f:if condition="{video.subtitles}">
+        <f:for each="{video.subtitles}" as="subtitle" iteration="iteration">
+            <track {f:if(condition: iteration.isFirst, then: 'default')} kind="subtitles" srclang="{subtitle.originalResource.properties.language}" src="{subtitle.originalResource.publicUrl}">
```

Comment:
@malteriechmann Two suggestions here:

1. Should we also add a select field for the [kind](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track#kind) attribute for greater flexibility? Possible values would be:
  * subtitles (Default)
  * captions
  * descriptions
  * chapters
  * metadata
2. Should we also add an optional additional input field for defining a [label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track#label) (as far as I understand this is displayed to the user to select the active subtitles)?

Reply:
> 2. Should we also add an optional additional input field for defining a [label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track#label) (as far as I understand this is displayed to the user to select the active subtitles)?

The browser will automatically use the value of the `srcland` attribute as the title, but to ensure proper recognition, for now I have included a label to indicate this to the browser.

```
    <track {f:if(condition: iteration.isFirst, then: 'default' )} kind="subtitles"
        label="{track.originalResource.properties.tx_html5mediakit_track_srclang}"
        srclang="{track.originalResource.properties.tx_html5mediakit_track_srclang}"
        src="{track.originalResource.publicUrl}">
```

Reply:
I am not so sure, if we should add a `kind` attribute, because the property itself is called `subtitles`.

Reply:
Maybe name the property `track` and then introduce `kind`.

Reply:
Thanks for the feedbacks again! now we have `label` and `kind` inputs as wished.